commands/ppreviewPush.go

```diff
@@ -125,12 +160,33 @@ func (args *PPreviewArgs) flags() []cli.Flag {
 	return flags
 }
 
+var _ = cmd(catMain, func() *cli.Command {
+	var args PPushArgs
+	return &cli.Command{
+		Name:  "push",
+		Usage: "identify changes to be made, and perform them",
+		Action: func(ctx *cli.Context) error {
+			if args.ConcurMode == "legacy" {
+				fmt.Fprint(os.Stderr, cmodewarn)
+				return exit(Push(args))
+			}
+			fmt.Fprintf(os.Stderr, "WARN: NO WARNING NEEDED (remove this before merge)\n")
```

Comment:
A GitHub comment so that `remove this before merge` action is guaranteed/cannot be forgotten.